genimage.c

```diff
@@ -93,6 +93,8 @@ static cfg_opt_t partition_opts[] = {
 	CFG_INT("partition-type", 0, CFGF_NONE),
 	CFG_BOOL("bootable", cfg_false, CFGF_NONE),
 	CFG_BOOL("read-only", cfg_false, CFGF_NONE),
+	CFG_BOOL("hidden", cfg_false, CFGF_NONE),
+	CFG_BOOL("dont-auto-mount", cfg_false, CFGF_NONE),
```

Comment:
This is still `dont-auto-mount`. So using `no-automount` below will fail.

Reply:
Right, thanks!